lib/default_cpolicy.json

```diff
@@ -0,0 +1,92 @@
+{
+	"rules": 
```

Comment:
why are we writing new config files in json? One of teh important reasons for adding scheme was NOT to manipulate the atomspace, but was to get away from the config files opencog.conf  To add et another config file in a new syntax that is opaque and not machine-controllable .. seems like a mistake to me.  We should not be proliferating, we should be reducing.